File: backend/databasehandler/src/main/java/com/pixtranslator/backend/databasehandler/Translateddictionary.java
package com.pixtranslator.backend.databasehandler;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Translateddictionary {

  @Id
  @GeneratedValue(strategy = GenerationType.AUTO)
  private long id;

  @Getter
  @Setter
  private String german;

  @Getter
  @Setter
  private String english;

}

File: backend/databasehandler/src/main/kotlin/com/pixtranslator/backend/databasehandler/Translateddictionary.kt
package com.pixtranslator.backend.databasehandler

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
class Translateddictionary(
  @Id
  @GeneratedValue(strategy = GenerationType.AUTO)
  private val id: Long = 0,

  private val german: String? = null,

  private val english: String? = null
)